source/_patterns/01-molecules/text/dropcap.twig

```diff
@@ -0,0 +1,45 @@
+<!-- ONLY FOR DEV TESTING -->
+<div class="text u-spacing--double">
+
+  <div>
+    <p class="has-drop-cap">"And this is <strong>another lorem</strong> Lorem ipsum dolor sit amet, consectetur
+      adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis
+      nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in
+      reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat
+      non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+  </div>
+
+  <div style="padding:5rem">
+    <p class="has-drop-cap"><strong>Freddie</strong> this is <strong>another lorem</strong> Lorem ipsum dolor sit amet,
+      consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim
+      veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor
+      in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat
+      non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+  </div>
+
+  <div class="has-drop-cap">
+    <p>"This is a sentence to test this" ... Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
+      tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco
+      laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
+      cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia
+      deserunt mollit anim id est laborum.</p>
+
+    <p>THIS</p>
+  </div>
+
+</div>
+
+<section class="l-grid l-grid--7-col u-shift--left--1-col--at-xxlarge l-grid-wrap--6-of-7 u-spacing--double--until-large  u-space--quad--top ">
+  <div class="c-article l-grid-item l-grid-item--l--4-col l-grid-item--xl--3-col">
+    <article class="text c-article__body u-spacing--double has-drop-cap">
+      <p><strong>Lorem ipsum dolor sit amet,</strong> consectetur adipiscing elit. <a href="">Donec malesuada</a>, est ut viverra euismod, dui dolor
```

Comment:
@davideleuterius I added another testing block that mimics what happens on an article. However I can't get the dropcap to apply here, though it does work on the main article pages now. Can you help me understand why it doesn't work on this example?

Reply:
@designerbrent 
I'll take a look at this today to see why this isn't applying as desired here.

Reply:
This is working correctly now.